src/components/Posts/Comment.js

```diff
@@ -1,64 +1,123 @@
-import React from 'react';
-import '../Posts/Comment.css'
+// import React from 'react';
```

Comment:
Just an fyi, you can safely delete things when you are using git, old code (once committed) can always be gotten back to :)